test/e2e/e2e_suite_test.go

```diff
@@ -224,14 +226,34 @@ func initBootstrapCluster(bootstrapClusterProxy framework.ClusterProxy, config *
 	By("Deploying cert-manager")
 	Expect(config.Variables).To(HaveKey(certManagerURL), "Missing %s variable in the config", certManagerURL)
 	certManagerComponentsUrl := config.GetVariable(certManagerURL)
-	certManagerResponce, err := http.Get(certManagerComponentsUrl) //use package "net/http"
+	certManagerResponse, err := http.Get(certManagerComponentsUrl) //use package "net/http"
 	Expect(err).ToNot(HaveOccurred(), "Failed to download cert-manager components from %s", certManagerComponentsUrl)
-	defer certManagerResponce.Body.Close()
+	defer certManagerResponse.Body.Close()
 
-	rawCertManagerReponse, err := io.ReadAll(certManagerResponce.Body)
+	rawCertManagerResponse, err := io.ReadAll(certManagerResponse.Body)
 	Expect(err).ToNot(HaveOccurred(), "Failed to read the cert-manager components file")
 
-	Expect(bootstrapClusterProxy.Apply(ctx, rawCertManagerReponse)).To(Succeed(), "Failed to apply cert-manager components to the bootstrap cluster")
+	Expect(bootstrapClusterProxy.Apply(ctx, rawCertManagerResponse)).To(Succeed(), "Failed to apply cert-manager components to the bootstrap cluster")
+
+	By("Waiting for cert manager to be available")
+	certManagerDeployments := []*appsv1.Deployment{
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: "cert-manager", Namespace: "cert-manager"},
```

Comment:
can you make deployment name/namespace constants? that way we can reuse and change them more easily

Reply:
Sure I will do it